lib/Db/ProvisioningMapper.php

```diff
@@ -138,4 +138,17 @@ public function get(int $id): ?Provisioning {
 			return null;
 		}
 	}
+
+	/**
+	 * @since 4.2.0
+	 */
+	public function uniqueImapHosts(): array {
+		$query = $this->db->getQueryBuilder();
+		$query->selectDistinct('imap_host')
+			->from('mail_provisionings');
+		$result = $query->executeQuery();
+		$data = $result->fetchAll(\PDO::FETCH_COLUMN);
+		$result->closeCursor();
+		return $data;
+	}
```

Comment:
^ as Codecov says, there is no test coverage. Please extend `\OCA\Mail\Tests\Integration\Db\ProvisioningMapperTest`